cmd/cueckoo/cmd/importpr.go

```diff
@@ -48,28 +55,84 @@ func importPRDef(c *Command, args []string) error {
 		return fmt.Errorf("%q is not a valid number", pr)
 	}
 
-	// TODO: validate that the number provided is indeed a valid PR
+	log.Printf("using github remote URL %q", cfg.githubURL)
 
-	// TODO: work out support for multiple PRs
+	branchName := fmt.Sprintf("importpr-%d", pr)
 
-	msg := fmt.Sprintf("Import PR %d from GitHub to Gerrit", pr)
-	payload, err := buildImportPRPayload(msg, importPRPayload{
-		PR: pr,
-	})
-	if err != nil {
+	// TODO: Note that mainErr's ctx is not wired here.
+	// We should wire it up and use it for e.g. FetchContext.
+	// For now, use a hard-coded timeout of 10s.
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	// If the branch already exists, refuse to continue.
+	if out, err := run(ctx,
+		"git", "show-ref", "--verify", "--quiet", fmt.Sprintf("refs/heads/%s", branchName),
+	); err == nil {
+		return fmt.Errorf("branch %q already exists; exiting. delete it to start over.", branchName)
```

Comment:
Would tend to agree with `staticcheck` here"

```
error strings should not end with punctuation or newlines (ST1005)
```

Reply:
yeah, I guess so. this is just an error that we print, and I wanted two sentences. I'll rewrite it to be a single sentence.